business/proxy_status.go

```diff
@@ -16,25 +16,35 @@ type ProxyStatusService struct {
 }
 
 func (in *ProxyStatusService) GetPodProxyStatus(cluster, ns, pod string) *models.ProxyStatus {
-	return castProxyStatus(kialiCache.GetPodProxyStatus(cluster, ns, pod))
+	return castProxyStatus(kialiCache.GetPodProxyStatus(cluster, ns, pod), false)
 }
 
-func castProxyStatus(ps *kubernetes.ProxyStatus) *models.ProxyStatus {
+func (in *ProxyStatusService) GetUpdatedPodProxyStatus(cluster, ns, pod string) *models.ProxyStatus {
+	return castProxyStatus(kialiCache.GetPodProxyStatus(cluster, ns, pod), true)
+}
```

Comment:
Right, I will make some changes. 

I didn't want to change the old method because I wasn't sure how to validate this in a sidecar. 
For waypoints, for example, and ingress/egress you can see how RDS is reported as IGNORED with `istioctl proxy-status`, but everything looks like SYNCED in case of the sidecars.